packages/tools/devtools/devtools-example/src/FluidObject.ts

```diff
@@ -95,7 +89,7 @@ export class AppData extends DataObject {
 			SharedCounter.getFactory(),
 			SharedMatrix.getFactory(),
 			SharedCell.getFactory(),
-			new SharedTreeFactory(),
+			SharedTree.getFactory(),
```

Comment:
Previously, SharedTree lacked 'getFactory()'.  Hence the direct use of the internal SharedTreeFactory type.